velodyne_pointcloud/include/velodyne_pointcloud/point_types.h

```diff
@@ -19,28 +19,39 @@
 #ifndef __VELODYNE_POINTCLOUD_POINT_TYPES_H
 #define __VELODYNE_POINTCLOUD_POINT_TYPES_H
 
-#include <pcl/point_types.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <sensor_msgs/point_cloud2_iterator.h>
 
 namespace velodyne_pointcloud
 {
   /** Euclidean Velodyne coordinate, including intensity and ring number. */
-  struct PointXYZIR
+/*  struct PointXYZIR
```

Comment:
right it is, and as mentioned before, relying on PCL for encoding means you'll need it for de-encoding. Plus the packing usually contains unused bit for memory alignment.
And Kinetic is coming so it could be a good time too :) Or at least, please close this PR if you deem it inappropriate.